src/ClientInterface.php

```diff
@@ -108,7 +108,7 @@
  * @method Status            ftaliasdel(string $alias)
  * @method Status            ftaliasupdate(string $alias, string $index)
  * @method Status            ftalter(string $index, Schema $schema, ?AlterArguments $arguments = null)
- * @method Status            ftcreate(string $index, Schema $schema, ?CreateArguments $arguments = null)
+ * @method Status            ftcreate(string $index, FieldInterface[] $schema, string $on = '', array $prefixes = [], string $filter = '', string $language = '', string $languageField = '', float $score = 0, string $scoreField = '', string $payloadField = '', bool $maxTextFields = false, int $temporary = 0, bool $noOffsets = false, bool $noHl = false, bool $noFields = false, bool $noFreqs = false, array $stopWords = [], bool $skipInitialScan = false)
```

Comment:
FieldInterface - why does this come before on?
The command order should follow FT.CREATE, even though most of the (post index) order is moveable.

Reply:
Also I think we should check the two hash field types - even picking a default (JSON?)

Reply:
In PHP pre-defined arguments cannot goes before required one, so since `$schema` is required it should goes before any pre-defined.

It checks field types within a builder, but it makes no sense to give it default value, because in this case we will always have `ON HASH` in argument set which makes no sense because it will be HASH by default anyway

Reply:
@chayim https://github.com/vladvildanov/predis/blob/vv-ftcreate-low-level/src/Command/Argument/Search/CreateArguments.php#L35